Revert to dev.63 SDK

isoos opened the pull request:
- runtime SDK is back at dev.63
- we download a separately tracked tool-env SDK for `analyzer`/`dartdoc` during docker image creation, set currently to dev.69
- tracking the two SDK separately, adjusted tests

/cc @sortie

mkustermann commented on app/lib/dartdoc/dartdoc_runner.dart:
```diff
@@ -63,8 +63,9 @@ class DartdocJobProcessor extends JobProcessor {
     await new Directory(outputDir).create(recursive: true);
 
     final toolEnv = await ToolEnvironment.create(
-      pubCacheDir: pubCacheDir,
```
Why was this removed?

isoos replied:
Not removed, moved 2 lines down.

mkustermann replied:
Right, appologies :)

isoos commented:
> I assume you were running a Dart 1 analyzer on the transitive code to ensure it's Dart 1 compatible?

Yes, although I missed the `pkg/_popularity` library, fixed it. Also fixed `dartfmt` formatting.

mkustermann commented:
Thanks, @isoos . Then go ahead and merge it, when it's ready. I'll make a new deployment afterwards.
